DependencyInjection/HttplugExtension.php

```diff
@@ -207,4 +217,20 @@ private function configureAuthentication(ContainerBuilder $container, array $con
                 ->addArgument(new Reference($authServiceKey));
         }
     }
+
+    /**
+     * Verifies that Puli bundle is installed.
+     *
+     * @param ContainerBuilder $container
+     *
+     * @throws \Exception
+     */
+    private function verifyPuliInstalled(ContainerBuilder $container)
+    {
+        if (false === $container->has('puli.discovery')) {
```

Comment:
Since `puli.discovery` is part of another bundle this if statement will always be true. This code has to be moved to a compiler pass, or you could check if the bundle is enabled.